view/next-project/src/pages/purchaseorders/index.tsx

```diff
@@ -135,9 +142,10 @@ export default function PurchaseOrders(props: Props) {
         <div className='mx-5 mt-10'>
           <div className='flex'>
             <Title title={'購入申請一覧'} />
-            <select className='w-100 '>
+            <select className='w-100 ' onChange={(e) => handleSelectedYear(Number(e.target.value))}>
               <option value='2021'>2021</option>
               <option value='2022'>2022</option>
+              <option value='2023'>2023</option>
```

Comment:
最新年度の反映&onChangeでの変更感謝！
objectじゃなくstringで管理するならこうなるかな
あとデフォルトで最新年度にしたい！
```suggestion
            <select className='w-100 ' onChange={(e) => setSelectedYear(e.target.value)}>
              <option value='2021'>2021</option>
              <option value='2022'>2022</option>
              <option value='2023' selected>2023</option>
```